thi-app/components/Sidebar.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+    View,
+    Text,
+    TouchableOpacity,
+    Image
+} from 'react-native';
+
+export default function Sidebar() {
+    return(
+        <View className="flex-1 justify-center bg-[#17468F] w-[339px] h-[834px]">
+            <View className="flex-1 w-[267px] h-[706px] self-center justify-evenly items-start flex-col">
+                <View className="flex-row justify-between items-center w-[267px] h-[45px]">
+                    <Image 
+                        style={{ width: 44, height: 45 }}
+                        source={require('../assets/images/thi_logo.png')}
+                    />
+                    <Text className="text-[#3BB4E4] font-sans font-bold text-[20px]">Texas Hearing Institute</Text>
+                </View>
+                <TouchableOpacity className="flex-row items-center">
+                    <Image source={require('../assets/images/home_icon.png')} className="w-6 h-6" />
+                    <Text className="text-white font-sans font-bold text-[20px] pl-2">Home</Text>
+                </TouchableOpacity>
+                <TouchableOpacity className="flex-row items-center">
+                    <Image source={require('../assets/images/students_icon.png')} className="w-6 h-6" />
+                    <Text className="text-white font-sans font-bold text-[20px] pl-2">Students</Text>
+                </TouchableOpacity>
+                <TouchableOpacity className="flex-row items-center">
+                    <Image source={require('../assets/images/games_icon.png')} className="w-6 h-6" />
+                    <Text className="text-white font-sans font-bold text-[20px] pl-2">Games</Text>
+                </TouchableOpacity>
+                <TouchableOpacity className="flex-row items-center">
+                    <Image source={require('../assets/images/timer_icon.png')} className="w-6 h-6" />
+                    <Text className="text-white font-sans font-bold text-[20px] pl-2">Timer</Text>
+                </TouchableOpacity>
+                <TouchableOpacity className="flex-row items-center">
+                    <Image source={require('../assets/images/settings_icon.png')} className="w-6 h-6" />
+                    <Text className="text-white font-sans font-bold text-[20px] pl-2">Settings</Text>
+                </TouchableOpacity>
```

Comment:
Can you add navigation to these buttons so that they'll jump to their respective pages when clicked. Refer to this: https://reactnavigation.org/docs/drawer-based-navigation